library/src/main/java/com/google/android/exoplayer2/audio/AudioTrack.java

```diff
@@ -443,6 +443,14 @@ public void configure(String mimeType, int channelCount, int sampleRate,
     }
 
     boolean passthrough = !MimeTypes.AUDIO_RAW.equals(mimeType);
+    if (Util.SDK_INT <= 23 && "foster".equals(Util.DEVICE) && "NVIDIA".equals(Util.MANUFACTURER)) {
+      if (channelConfig == (AudioFormat.CHANNEL_OUT_5POINT1 | AudioFormat.CHANNEL_OUT_BACK_CENTER) )
```

Comment:
Changed.  Just went with that for explicit intent, but the change is easier to read and as easy to decipher.